resources/i18n/ko.json

```
@@ -1089,7 +1089,9 @@
     "DescThemeMode": "Backend.AI WebUI가 장치에서 어떻게 보이는지 설정합니다.",
     "ThemeMode": "화면 모드",
     "BootstrapScriptEmpty": "스크립트가 비어있습니다. 스크립트를 입력해 주세요.",
-    "BootstrapScriptDeleted": "부트스트랩 스크립트가 삭제되었습니다."
+    "BootstrapScriptDeleted": "부트스트랩 스크립트가 삭제되었습니다.",
+    "ClassicSessionLauncher": "Classic 세션 런처",
+    "DescClassicSessionLauncher": "최신 세션 런처 대신 Classic 세션 런처를 사용합니다. Classic 세션 런처에는 최신 기능이 존재하지 않을 수 있습니다."
```

Same as here: feature comparison has to be specific. 
'Latest' is a status, so calling it as 'Classic vs Neo' is more suitable.

```suggestion
    "DescClassicSessionLauncher": "최신의 Neo 세션 런처 대신 Classic 세션 런처를 사용합니다. Classic 세션 런처에는 최신 기능이 지원되지 않을 수 있습니다."
```